wolfhsm/wh_client_crypto.h

```diff
@@ -150,57 +150,55 @@ int wh_Client_MakeExportCurve25519Key(whClientContext* ctx,
  * @param[in] keyId Key ID to be associated with the Ecc key.
  * @return int Returns 0 on success or a negative error code on failure.
  */
-int wh_Client_SetKeyIdEcc(ecc_key* key, whNvmId keyId);
+int wh_Client_EccSetKeyId(ecc_key* key, whKeyId keyId);
 
 /**
  * @brief Gets the wolfHSM keyId being used by the wolfCrypt struct.
  *
  * This function gets the device context of a Ecc key that was previously
- * set by either the crypto callback layer or wh_Client_SetKeyIdEcc.
+ * set by either the crypto callback layer or wh_Client_EccSetKeyId.
  *
  * @param[in] key Pointer to the Ecc key structure.
  * @param[out] outId Pointer to the key ID to return.
  * @return int Returns 0 on success or a negative error code on failure.
  */
-int wh_Client_GetKeyIdEcc(ecc_key* key, whNvmId* outId);
+int wh_Client_EccGetKeyId(ecc_key* key, whKeyId* outId);
 
+/* TODO: Send key to server */
```

Comment:
These functions are marked as TODO but aren't anymore, right? If so, remove in next PR or in the pending doxygen update PR